publons/PublonsPlugin.inc.php

```diff
@@ -52,7 +52,7 @@ function getName() {
     /**
      * Get the display name of this plugin
      * @return string
-     * @see PKPPlugin::getDisplayName()
+     * @see Plugin::getDisplayName()
```

Comment:
Plugin used to extend from PKPPlugin, but not any more
the definition for 2.4.8 is here: https://pkp.sfu.ca/ojs/doxygen/stable/html/classPlugin.html
the definition for 3.0.0 is here: https://pkp.sfu.ca/ojs/doxygen/master/html/classPlugin.html